app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,145 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence.
+  </p>
+
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "quote",
+      colour: "yellow",
+      image_1_path: "static/images/content/hero-images/0012.jpg",
+      image_1_alt: "Pupils listening to a science lesson.",
+      image_2_path: "static/images/content/hero-images/0007.jpg",
+      image_2_alt: "Teacher talking to a pupil in a classroom.",
+    ) do %>
+       <p>
+quote  </p>
+
+  <p>
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your pupils' ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover include:
+      </p>
+      <p>
+         Themes you’ll cover for 11 to 14 year olds (key stage 3):
```

Comment:
@caitlin-reidy same as maths it would be good to get some practical examples of how these themes translate into the classroom and their teaching.